angular/projects/spark-angular/src/lib/components/sprk-icon/sprk-icon.component.spec.ts

```diff
@@ -31,17 +31,52 @@ describe('SprkIconComponent', () => {
     expect(component.getClasses()).toEqual('sprk-c-Icon');
   });
 
+  it('should add the correct icon based on the icon type set', () => {
+    component.iconName = 'bell';
+    fixture.detectChanges();
+    expect(
+      iconElement.querySelector('use').hasAttribute('xlink:href'),
+    ).toBeTruthy();
+    expect(iconElement.querySelector('use').getAttribute('xlink:href')).toEqual(
+      '#bell',
+    );
+  });
+
+  // TODO: deprecate iconType
   it('should add the correct icon based on the icon type set', () => {
     component.iconType = 'bell';
     fixture.detectChanges();
     expect(
-      iconElement.querySelector('use').hasAttribute('xlink:href')
+      iconElement.querySelector('use').hasAttribute('xlink:href'),
     ).toBeTruthy();
     expect(iconElement.querySelector('use').getAttribute('xlink:href')).toEqual(
-      '#bell'
+      '#bell',
     );
   });
 
+  it('should add the default viewBox if not set', () => {
+    component.iconType = 'bell';
+    fixture.detectChanges();
+    expect(iconElement.hasAttribute('viewBox')).toBeTruthy();
+    expect(iconElement.getAttribute('viewBox')).toEqual('0 0 64 64');
+  });
+
+  it('should add the default viewBox if not set', () => {
```

Comment:
Need to update the name of this test.

Reply:
✅ fixed this and pushed up

Reply:
I think these names need to be reversed, since the default is '0 0 64 64'

Reply:
agree, I changed it in the other test that's why this comment isn't marked as "outdated"

Reply:
my bad, you're right. I just needed to slow down lol

Reply:
pushed up!